docs/Contributing.md

```diff
@@ -70,6 +85,12 @@ Which will also resolve the relevant static content paths for serving HTTP.
 
 # Running Unit Tests
 
+If your version of Go < 1.13, you have to run:
```

Comment:
Yes, you are right, let's move this to the beginning of the "Hacking" section. E.g.:
```
First, you'll need Go [(version 1.11.x or greater)](https://golang.org/doc/install) ...

If your version of Go < 1.13, you have to run:
```

Also, it looks like the whole section needs an update regarding Go modules. But that's for a separate PR.